source/xrefdelta.tex

```diff
@@ -110,5 +110,9 @@
 \movedxref{expos.only.func}{expos.only.entity}
 \removedxref{expos.only.types}
 
+% P2614R2 Deprecate numeric_limits::has_denorm
+\movedxref{denorm.style}{depr.numeric.limits.has.denorm}
+\removedxref{fp.style}
```

Comment:
How about having an "split reference" command. Whereas it is true that [fp.style] was removed, it seems more useful to say it was split between [round.style] and [depr.numeric.limits.has.denorm].

Reply:
What should the representation in the actual index be?

Reply:
A combination of these:
![1677122412](https://user-images.githubusercontent.com/21071787/220814379-3111414b-e71d-4ee7-bc1a-35ef657be4cc.png)
So something like
> fp.style _split_, _see_ round.style, depr.numeric.limits.has.denorm

Reply:
Pull requests welcome :-)